src/ralph/backends/data/async_es.py

```diff
@@ -234,6 +236,10 @@ async def write(  # noqa: PLR0913
             operation_type (BaseOperationType or None): The mode of the write operation.
                 If `operation_type` is `None`, the `default_operation_type` is used
                 instead. See `BaseOperationType`.
+            simultaneous (bool): If `True`, chunks will be written concurrently.
+                If `False` (default), chunks will be written sequentially.
+            max_num_simultaneous (int or None): If simultaneous is `True`, the maximum
+                number of chunks to write concurrently. If `None` it defaults to 1.
```

Comment:
Maybe we could remove the `simultaneous` parameter ? and only have `max_num_simultaneous` (default: 1). If `None`, there would be no maximum ?

Reply:
Great idea) How about renaming it to `concurrency=1`?
I'm hesitating about allowing unlimited concurrency - wouldn't it be problematic if the `data` is unbound?

Reply:
Agreed, unlimited concurrency is probably not a good idea.

`concurency=1` is fine with me !

Reply:
Updated with `concurrency`)